handlers/api_eth1.go

```diff
@@ -192,6 +192,23 @@ func ApiETH1AccountProducedBlocks(w http.ResponseWriter, r *http.Request) {
 		}
 	}
 
+	if isSortAsc {
+		// Remove duplicates from the block list
+		allKeys := make(map[uint64]bool)
+		list := []uint64{}
+		for _, item := range blockList {
+			if _, ok := allKeys[item]; !ok {
+				allKeys[item] = true
+				list = append(list, item)
+			}
+		}
+		blockList = list
+
+		// Trim to the blocks that are within the limit range
+		sort.Slice(blockList, func(i, j int) bool { return blockList[i] < blockList[j] })
+		blockList = blockList[:limit]
+	}
```

Comment:
**Minor:** As discussed, personally I think it is confusing to only to this for asc and I would suggest to do it all the time. This seems less error prone in the future, does it not?